src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --color-nav-selected: #00acdc;
+  --header-height: 60px;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+  align-content: center;
+  position: relative;
+  height: var(--header-height);
```

Comment:
it's not a good practice to hardcode height for elements with content inside. just add `padding: 23px 0;` to your links 
```suggestion
```